src/arch/x86_64/encodings.zig

```diff
@@ -1,2624 +0,0 @@
-const Encoding = @import("Encoding.zig");
-const Mnemonic = Encoding.Mnemonic;
-const OpEn = Encoding.OpEn;
-const Op = Encoding.Op;
-const Mode = Encoding.Mode;
-const Feature = Encoding.Feature;
-
-const modrm_ext = u3;
-
-pub const Entry = struct { Mnemonic, OpEn, []const Op, []const u8, modrm_ext, Mode, Feature };
-
-// TODO move this into a .zon file when Zig is capable of importing .zon files
```

Comment:
great job, you've earned 1 free "TODO" comment

Reply:
i will use it wisely